dlk/python/dlk/plugins/tf.py

```diff
@@ -1168,6 +1168,20 @@ def infer_dtype() -> DataType:
                 input_ops,
                 dimension_format=current_format,
             )
+        elif op_type == 'Mean':
+            keepdims = node.attribute('keep_dims')[0]
+            if not shape:
+                attributes = {'keep_dims': keepdims}
+                shape = infer_shape(attributes)
```

Comment:
Does `keep_dims` change this `shape` correctly ?
We don't need to implement `infer_shape()` in `Mean(Operator)` ?

Reply:
The `shape` is directly obtained from .pb graph, `keep_dims` doesn't change the `shape` of the op. I will add the `infer_shape` in next commit.

Reply:
thanks 👍